src/config/constants.js

```diff
@@ -39,7 +39,8 @@ export const BLOCK_MATH_REGEX = /(\\\[(.*?)\\])/g;
 export const INLINE_MATH_REGEX = /(\\\((.*?)\\\))/g;
 export const RADIX = 10;
 
-export const FILTER_ALL_SPACE_ID = 'defaultid';
+export const FILTER_ALL_SPACES_ID = 'allSpacesSelectId';
```

Comment:
Probably want to call this the opposite. `SELECT_ALL_SPACES_ID` or `NO_SPACE_FILTER_ID`. Same with users.